.travis.yml

```diff
@@ -21,55 +21,46 @@ env:
     - RUST_NIGHTLY_VERSION=nightly-2018-06-29
     - RUST_CLIPPY_VERSION=0.0.211
     - EJB_RUST_BUILD_DIR="$TRAVIS_BUILD_DIR/exonum-java-binding-core/rust/"
+    - SODIUM_BUILD=1
+    - ROCKSDB_BUILD=1
+    - SNAPPY_BUILD=1
+    - SKIP_COMPILE=true
+  matrix:
+    - CHECK_RUST=true
+    - CHECK_RUST=false
+# force building instead of using from apt.
+
+# run java tests for jdk8&jdk10 without rust checks, 
+# check rust code with jdk8 without running java tests
```

Comment:
As far as i can see, we're running both jdk8 and jdk10 with java tests, but disabling rust checks on jdk10. So these two comment lines need rewording at least.